src/server.rs

```diff
@@ -1,21 +1,19 @@
 pub mod error;
 
-use std::net::{IpAddr, Ipv4Addr, SocketAddr, TcpListener};
+use std::net::TcpListener;
```

Comment:
Consider moving this to `server` and renaming the file to `mod.rs`.

Reply:
I'm actually more partial to self named mod files and I'm trying to progressively move away from mod.rs

Reply:
> I'm actually more partial to self named mod files and I'm trying to progressively move away from mod.rs

Oh yeah? Why do you prefer that? I sorta always liked the ‘mod.rs’ method for how it contains a module inside a directory.